smart-metastore/src/main/java/org/smartdata/metastore/dao/AccessEventAggregator.java

```diff
@@ -86,57 +93,57 @@ private void createTable() {
       LOG.error("Create table error: " + table, e);
       return;
     }
-    if (!eventBuffer.isEmpty() || !lastAccessCount.isEmpty()) {
-      Map<String, Integer> accessCount = this.getAccessCountMap(eventBuffer);
-      Set<String> now = new HashSet<>(accessCount.keySet());
-      accessCount = mergeMap(accessCount, lastAccessCount);
+
+    if (!eventBuffer.isEmpty() || !unmergedAccessCounts.isEmpty()) {
```

Comment:
The `if (!eventBuffer.isEmpty() || !unmergedAccessCounts.isEmpty())` condition contains a large block of code. For readablity sake, it could be useful to break this up into smaller methods that each handle a part of the functionality within the block for better readability and maintainability.